app/scripts/modules/core/src/presentation/icons/vectors/cluster.svg

```diff
@@ -1,3 +1,14 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-  <path d="M7 4v3H4V4h3zm0-1H4c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1V4c0-.6-.4-1-1-1zM14 4v3h-3V4h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1V4c0-.6-.4-1-1-1zM21 4v3h-3V4h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1V4c0-.6-.4-1-1-1zM7 11v3H4v-3h3zm0-1H4c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1zM14 11v3h-3v-3h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1zM21 11v3h-3v-3h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1zM7 18v3H4v-3h3zm0-1H4c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1zM14 18v3h-3v-3h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1zM21 18v3h-3v-3h3zm0-1h-3c-.6 0-1 .4-1 1v3c0 .6.4 1 1 1h3c.6 0 1-.4 1-1v-3c0-.6-.4-1-1-1z"/>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
+<path d="M5,1v4H1V1H5z M5,0H1C0.4,0,0,0.4,0,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1V1C6,0.4,5.6,0,5,0z M5,10v4H1v-4H5z M5,9H1
+	c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C6,9.4,5.6,9,5,9z M5,19v4H1v-4H5z M5,18H1c-0.6,0-1,0.4-1,1v4
+	c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C6,18.4,5.6,18,5,18z"/>
+<path d="M14,1v4h-4V1H14z M14,0h-4C9.4,0,9,0.4,9,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1V1C15,0.4,14.6,0,14,0z M14,10v4h-4v-4H14z
+	 M14,9h-4c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C15,9.4,14.6,9,14,9z M14,19v4h-4v-4H14z M14,18h-4
+	c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C15,18.4,14.6,18,14,18z"/>
+<path d="M23,1v4h-4V1H23z M23,0h-4c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1V1C24,0.4,23.6,0,23,0z M23,10v4h-4v-4H23z
+	 M23,9h-4c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C24,9.4,23.6,9,23,9z M23,19v4h-4v-4H23z M23,18h-4
+	c-0.6,0-1,0.4-1,1v4c0,0.6,0.4,1,1,1h4c0.6,0,1-0.4,1-1v-4C24,18.4,23.6,18,23,18z"/>
```

Comment:
This now takes up the full bounding box. It might feel a little tight, let's check it out together once its in the MD view.

Reply:
woohoo! thank you, i'll see how it looks